main/boost/APKBUILD

```diff
@@ -46,27 +46,35 @@ _libs="
 	filesystem
 	graph
 	iostreams
+	locale
+	log_setup
+	log
 	math
 	prg_exec_monitor
 	program_options
 	python3
-	python
+	python2
 	random
 	regex
 	serialization
 	signals
+	stacktrace_basic
+	stacktrace_noop
 	system
 	thread
+	timer
+	type_erasure
 	unit_test_framework
 	wave
 	wserialization
 	"
 for _lib in $_libs; do
-	case "$_options_carch" in
-		*$_lib*) continue;;
-	esac
+	for _option in $_options_carch; do
+		case "$_option" in --without-${_lib}) continue ;; esac
+	done
```

Comment:
I think the logic here is broken. There is a nested loop here and the `continue` statement there will only continue the the inner loop.

Reply:
I concur. Moving the assignment into the inner loop looks to be what we want.